lib/ontologies_linked_data/models/agents/agent.rb

```
@@ -26,7 +26,13 @@ class Agent < LinkedData::Models::Base
       enable_indexing(:agents_metadata)
 
       def embedded_doc
-        "#{self.name} #{self.acronym} #{self.email} #{self.agentType}"
+        {
+          "id": "#{self.id}",
```

RuboCop (1.73)

[convention] Prefer `to_s` over string interpolation.

(Style/RedundantInterpolation)